src/parameter.h

```diff
@@ -35,6 +36,8 @@ class Parameter final : public KeyValueStore::Service {
   private:
     Status GetValues(ServerContext* context, ServerReaderWriter<Response, Request>* stream);
 
+    AXParameter* ax_parameter;
+    GError* _error;
```

Comment:
There is no need for this to be a class member. It can be a local variable in each function it is used.